packages/pug-load/test/__snapshots__/index.test.js.snap

```diff
@@ -122,6 +154,43 @@ Object {
                   "fullPath": "<dirname>/script.js",
                   "line": 6,
                   "path": "script.js",
+                  "raw": Object {
+                    "data": Array [
+                      100,
+                      111,
+                      99,
+                      117,
+                      109,
+                      101,
+                      110,
+                      116,
+                      46,
+                      119,
+                      114,
+                      105,
+                      116,
+                      101,
+                      40,
+                      39,
+                      104,
+                      101,
+                      108,
+                      108,
+                      111,
+                      32,
+                      119,
+                      111,
+                      114,
+                      108,
+                      100,
+                      33,
+                      39,
+                      41,
+                      59,
+                      10,
+                    ],
+                    "type": "Buffer",
```

Comment:
You can customise how jest renders certain values in the snapshot via serializers. You can see an example in: https://github.com/pugjs/pug/blob/master/scripts/filename-serializer.js

I suggest we add one for `Buffer`. For now it could just call `.toString('utf8')` and tag the result as a Buffer. If we can have it detect Buffers that are probably binary data rather than text data, we could also have it output those as something like `Buffer(SHA_HASH)` so they're shorter and easier to review in the snapshots.

Reply:
Ah, that's nice. I was wondering that, but figured for the small buffers that existed, it wasn't too bad.

Reply:
425f44a